File: src/main/java/frc/team4069/robot/commands/vacuum/StopVacuumCommand.java
package frc.team4069.robot.commands.vacuum;

import frc.team4069.robot.commands.CommandBase;

public class StopVacuumCommand extends CommandBase {

    public StopVacuumCommand() {
        requires(vacuum);
    }

    @Override
    protected void initialize() {
        vacuum.stop();
    }

    @Override
    protected boolean isFinished() {
        return true;
    }
}

File: src/main/java/frc/team4069/robot/commands/vacuum/StopVacuumCommand.kt
package frc.team4069.robot.commands.vacuum

import edu.wpi.first.wpilibj.command.InstantCommand
import frc.team4069.robot.subsystems.VacuumSubsystem

class StopVacuumCommand : InstantCommand() {
    init {
        requires(VacuumSubsystem)
    }

    override fun initialize() = VacuumSubsystem.stop()
}